x/capability/keeper/keeper_test.go

```diff
@@ -35,19 +34,20 @@ func (suite *KeeperTestSuite) SetupTest() {
 
 	suite.ctx = app.BaseApp.NewContext(checkTx, abci.Header{Height: 1})
 	suite.keeper = keeper
-	suite.app = app
 }
 
 func (suite *KeeperTestSuite) TestInitializeAndSeal() {
 	sk := suite.keeper.ScopeToModule(bank.ModuleName)
 
 	caps := make([]*types.Capability, 5)
+	// Get Latest Index before creating new ones to sychronize indices correctly
+	prevIndex := suite.keeper.GetLatestIndex(suite.ctx)
```

Comment:
Needed to grab the index we are at before adding new ones because `InitChain` of simapp will create capabilities now that ibc app modules like `transfer` bind to ports on InitChain.